core/templates/ide/clion/CMakeLists.txt.tpl

```diff
@@ -0,0 +1,50 @@
+set(WCOSA_CMD "python {{wcosa-path}}/core/wcosa.py")
```

Comment:
`.tpl` vs `.tmp`? why not `.tmpl` and mix 'em both together

Reply:
Oops I messed up the names, will fix the extensions